add mainnet docs (#180)
ObolNetwork/obol-docs: docs/int/quickstart/quickstart-alone.md

docker compose up -d --build
```

> ## Run mainnet node

Reviewer: ```suggestion
## Run a DV on mainnet
```

> This section is intended for users who wish to run their node on Ethereum mainnet.

Reviewer: ```suggestion
This section is intended for users who wish to run their distributed validator on Ethereum mainnet. 
```

1. First make sure that your DV stack is [up-to-date](./quickstart-alone#updating-dv-stack).
2. Then, copy the `.env.sample` file to `.env`
```

> cp .env.sample .env

Reviewer 2: suggest `cp -n .env.sample .env` to avoid overriting existing file

```
3. In your `.env` file, uncomment and set values for `NETWORK` & `LIGHTHOUSE_CHECKPOINT_SYNC_URL`
```
...
# Overrides network for all the relevant services.
NETWORK=mainnet
...

> # Checkpoint sync url used by lighthouse to fast sync.

Reviewer 2: add a link where they can find these

Author: cool, added https://eth-clients.github.io/checkpoint-sync-endpoints/#mainnet

LIGHTHOUSE_CHECKPOINT_SYNC_URL=https://mainnet.checkpoint.sigp.io/https://eth-clients.github.io/checkpoint-sync-endpoints/#mainnet
...
```

Your DV stack is now mainnet ready 🎉

#### Remote mainnet beacon node

If you already have a mainnet beacon node running somewhere and you want to use that instead of running EL (`geth`) & CL (`lighthouse`) as part
of the repo, you can disable these images. To do so, follow these steps:

1. Copy the `docker-compose.override.yml.sample` file
```

> cp docker-compose.override.yml.sample docker-compose.override.yml

Reviewer 2: `cp -n ...`

```
2. Uncomment the `profiles: [disable]` section for both `geth` and `lighthouse`. The override file should now look like this
```markdown
services:
  geth:
    # Disable geth
    profiles: [disable]
    # Bind geth internal ports to host ports
    #ports:
      #- 8545:8545 # JSON-RPC
      #- 8551:8551 # AUTH-RPC
      #- 6060:6060 # Metrics

  lighthouse:
    # Disable lighthouse
    profiles: [disable]
    # Bind lighthouse internal ports to host ports
    #ports:
      #- 5052:5052 # HTTP
      #- 5054:5054 # Metrics
...
```
3. Then, uncomment and set the `CHARON_BEACON_NODE_ENDPOINTS` variable in the `.env` file to your mainnet beacon node's URL
```markdown
...
# Connect to one or more external beacon nodes. Use a comma separated list excluding spaces.

> CHARON_BEACON_NODE_ENDPOINTS=

Reviewer 2: add: `CHARON_BEACON_NODE_ENDPOINTS=<your remove beacon node address>`

Author: cool, added `CHARON_BEACON_NODE_ENDPOINTS=<YOUR_REMOTE_MAINNET_BEACON_NODE_URL>`

> ...
> # Connect to one or more external beacon nodes. Use a comma separated list excluding spaces.
> CHARON_BEACON_NODE_ENDPOINTS=
> ...
> ```

Reviewer 3: ```suggestion
...
# Connect to one or more external beacon nodes. Use a comma separated list excluding spaces.
CHARON_BEACON_NODE_ENDPOINTS=http://YOUR_MAINNET_BEACON_NODE_URL/
...
```
```

#### Mainnet node with mev-boost

If you are running your mainnet DV node with `mev-boost`, you need to uncomment and set the `MEVBOOST_RELAYS` variable in the `.env` file
```markdown
...
# MEV-Boost docker container image version, e.g. `latest` or `v1.4.0`.
#MEVBOOST_VERSION=
MEVBOOST_RELAYS=https://0x9000009807ed12c1f08bf4e81c6da3ba8e3fc3d953898ce0102433094e5f22f21102ec057841fcb81978ed1ea0fa8246@builder-relay-mainnet.blocknative.com
...
```
You can also use the [flashbots relay](https://boost-relay.flashbots.net/).

> #### Voluntary exit mainnet validator

Reviewer: ```suggestion
#### Exit a mainnet distributed validator
```

If you want to exit your mainnet validator, you need to uncomment and set the `EXIT_EPOCH` variable in the `.env` file

```markdown
...
# Cluster wide consistent exit epoch. Set to latest for fork version, see `curl $BEACON_NODE/eth/v1/config/fork_schedule`

> EXIT_EPOCH=194048

Reviewer 2: isn't this in the future? We should pick something in the past otherwise people will run it soon and it will not work

> EXIT_EPOCH=194048

Reviewer 3: I'd suggest to not include directly rather specify this is capella epoch in parenthesis

Author: cool, modified to this
```
...
# Currently, the latest fork is bellatrix (epoch: 144896)
EXIT_EPOCH=144896
```
